spk/shairport/src/dsm-control.sh

```diff
@@ -0,0 +1,96 @@
+#!/bin/sh
+
+# Package
+PACKAGE="shairport"
+DNAME="ShairPort"
+# Others
+SHAIRPORT="${SYNOPKG_PKGDEST}/bin/shairport"
+VAR_DIR="${SYNOPKG_PKGDEST}/var"
+PID_FILE="${VAR_DIR}/${PACKAGE}.pid"
+
+
+start_daemon ()
+{
+    # Launch the service in the background.
+	cd ${SYNOPKG_PKGDEST}
+	$SHAIRPORT --daemon --pidfile="$PID_FILE" --log=$SYNOPKG_TEMP_LOGFILE
```

Comment:
- Use `${VARIABLE}` for consistency. 
- The logfile should be placed in `${INSTALL_DIR}/var/shairport.log` or something like that.